Igalvan - Livingston

```diff
@@ -1,5 +1,6 @@
 Livingston
 Restaurants
+Mayrita's 
 Things to do
```

Comment:
@igalvan Try this out. You can drag an image to this form and it gets uploaded to github 😃 
<img width="321" alt="screen shot 2016-06-22 at 2 15 59 am" src="https://cloud.githubusercontent.com/assets/2144028/16241216/4e145fe0-381f-11e6-83e3-52bd9f82a71f.png">